README.md

```diff
@@ -1,3 +1,31 @@
+# Support for Sidecar
+
+This repo contains a fork of Intel's SDE for Tofino.  It contains modifications
+to support building and running on Helios (Oxide's illumos distribution) and the
+Sidecar switch.
+
+The upstream SDE uses a suite of Python scripts to configure and build for a
+wide variety of Linux distros and Tofino family asics.  To build and package
+this fork you should simply run:
+
+```
+tofino-sde$ export SDE=`pwd`
+tofino-sde$ ./oxide/build.sh
+[lots of output]
+tofino-sde$ ./oxide/pakage.sh
+[lots of output]
+```
+
```

Comment:
In `bf_sde` we would specify a version to build, is that an option here as well?